phpunit.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.4/phpunit.xsd" bootstrap="vendor/autoload.php" colors="true" cacheDirectory=".phpunit.cache">
+  <testsuites>
+    <!--<testsuite name="Unit">
```

Comment:
If you remove the UnitTest, this should also be removed. But as mentioned above, I'll also write something about this in the UnitTest, so it's best to read that first.